TestProject.Spec/StepDefinitions/DynamicControlsTestStepDefinitions.cs

```diff
@@ -0,0 +1,71 @@
+﻿using NUnit.Framework;
+using TestProject.Tests.Pages;
+using TestProject.Utils;
+
+namespace TestProject.Spec.StepDefinitions
+{
+    [Binding]
+    public class DynamicControlsTestStepDefinitions
+    {
+        private readonly ScenarioContext _scenarioContext;
+
+        public DynamicControlsTestStepDefinitions(ScenarioContext scenarioContext)
+        {
+            _scenarioContext = scenarioContext;
+        }
+
+        [BeforeScenario("UI")]
+        public static void Setup()
+        {
+            Browser.GetDriver().Navigate().GoToUrl(ConfigReader.GetConfigValue("webUIUrl"));
+        }
+
+        [AfterScenario("UI")]
+        public static void TearDown()
+        {
+            Browser.GetDriver().Quit();
+        }
+
+        [Given("I have opened the Dynamic Control page")]
+        public void GivenIHaveOpenedTheDynamicControlPage()
+        {
+            var mainPage = new MainPage();
+            mainPage.ClickOnDynamicControl();
+            _scenarioContext["mainPage"] = mainPage;
+        }
+
+        [Given("I click the enable button")]
+        public void WhenIClickOnTheEnableButton()
+        {
+            var dynamicControlsPage = new DynamicControlsPage();
+            dynamicControlsPage.ClickOnEnableButton();
+            _scenarioContext["dynamicControlsPage"] = dynamicControlsPage;
```

Comment:
page won't change, we can save it to the class field (it always acts the same, so it won't affect us in any way)